parsers/dataField/utils.go

```diff
@@ -37,3 +40,19 @@ func extractTokenAndNonce(arg []byte) (string, uint64) {
 
 	return string(identifier), nonce.Uint64()
 }
+
+func isEmptyAddr(pubKeyConverter core.PubkeyConverter, receiver []byte) bool {
+	emptyAddr := make([]byte, pubKeyConverter.Len())
```

Comment:
Do we really need to pass the pub key converter here just to get the pub key length?
Why not just use `AddressLen` as input param in args for `operationDataFieldParser`?